x-pack/filebeat/module/o365/audit/test/22-yammer.log-expected.json

```diff
@@ -0,0 +1,103 @@
+[
+    {
+        "@timestamp": "2020-02-28T09:42:45.000Z",
+        "client.address": "79.159.10.151:12345",
+        "client.ip": "79.159.10.151",
+        "client.port": "12345",
+        "event.action": "GroupCreation",
+        "event.category": "iam",
+        "event.code": "Yammer",
+        "event.dataset": "o365.audit",
+        "event.id": "2af7bbf1-d5d8-5cb0-8aca-f4ad8a087594",
+        "event.kind": "event",
+        "event.module": "o365",
+        "event.outcome": "success",
+        "event.provider": "Yammer",
+        "event.type": "group",
```

Comment:
would it be possible to have "creation" added to event.type.  So it would be event.cateogory = iam, event.type = "creation, group"